src/script/descriptor.cpp

```diff
@@ -200,6 +200,12 @@ struct PubkeyProvider
 
     /** Derive a private key, if private data is available in arg. */
     virtual bool GetPrivKey(int pos, const SigningProvider& arg, CKey& key) const = 0;
+
+    /** Return all (extended public keys for this PubkeyProvider
+     * param[out] pubkeys Any public keys
+     * param[out] ext_pubs Any extended public keys
+     */
+    virtual void GetRootPubkey(std::set<CPubKey>& pubkeys, std::set<CExtPubKey>& ext_pubs) const = 0;
```

Comment:
3d4e3b9cf3801a93d1a4af62994900b4f69f2edd
I suggest renaming instances of `Pubkey` to `PubKey` to be consistent with `*PrivKey` functions.
```suggestion
    virtual void GetRootPubKey(std::set<CPubKey>& pubkeys, std::set<CExtPubKey>& ext_pubs) const = 0;
```

Reply:
Done